lib/arel_extensions/math.rb

```diff
@@ -40,7 +40,7 @@ def +(other)
       when Arel::Nodes::Function
         Arel.grouping(Arel::Nodes::Addition.new self, other)
       else
-        col = self.respond_to?(:relation)? ArelExtensions::column_of(self.relation.table_name, self.name.to_s) : nil
+        col = self.respond_to?(:relation)? ArelExtensions.column_of(self.relation.table_name, self.name.to_s) : nil
```

Comment:
I would personally prefer the short version, `Arel.column_of`.  But Julien might disagree with me.

Reply:
Also, that's the kind of place where, IMHO, in Ruby one would favor

```ruby
col = ArelExtensions.column_of(self.relation.table_name, self.name.to_s) if self.respond_to?(:relation)?
```

Reply:
It is defined to be used.